helix-term/src/commands.rs

```diff
@@ -1019,6 +1019,34 @@ mod cmd {
         doc.format(view.id)
     }
 
+    fn earlier(editor: &mut Editor, args: &[&str], event: PromptEvent) {
+        let sotp = match args.join(" ").parse::<helix_core::StepsOrTimePeriod>() {
+            Ok(sotp) => sotp,
+            Err(msg) => {
+                editor.set_error(msg);
+                return;
+            }
+        };
+        let view_id = editor.view().id;
```

Comment:
You can call `let (view, doc) = editor.current()` for this

Reply:
Done.